stories/input.js

```diff
@@ -14,12 +14,39 @@ import {
   TextBody,
   TextSmall,
   Tooltip,
+  SingleLineInputBase,
 } from '../components';
 import { IconCalendarSmallOutline, IconInfoBadgedSmallFilled } from '@teamleader/ui-icons';
 
 const colors = ['aqua', 'gold', 'mint', 'neutral', 'ruby', 'teal', 'violet'];
-const sizes = ['small', 'medium', 'large'];
+const elements = [null, 'input', 'textarea'];
+const sizes = [null, 'small', 'medium', 'large'];
```

Comment:
Q: Not really sure why null is being added here? 😬

Reply:
It is for the storybook `select` knob.
Adding `null` tot those arrays, combined with writing the prop on the component like so:
```
size={select('size', sizes) || undefined}
```
Simplifies the 'Story Source' (the code snippet you can copy-paste)

Reply:
But has no use anymore, as I removed the story, so removing it too 👍.